style.css

```diff
@@ -1,3 +1,13 @@
+:root {
+    --text-color: #121218;
+    --bg-color: #F2F3F8;
+}
+
+[data-theme="dark"] {
+    --text-color: #F2F3F8;
+    --bg-color: #202128;
+}
```

Comment:
Good use of variables here, though I would have also picked new colors for everything else because they look too bright against the dark background.